cmd/root.go

```diff
@@ -114,6 +119,10 @@ any client SSL certificates.`,
 		"Path to a service account key to use for authentication.")
 	cmd.PersistentFlags().BoolVarP(&c.conf.GcloudAuth, "gcloud-auth", "g", false,
 		"Use gcloud's user configuration to retrieve a token for authentication.")
+	cmd.PersistentFlags().StringVarP(&c.telemetryProject, "telemetry-project", "j", "",
+		"Enable telemetry with the provided project ID to use for metrics and traces.")
```

Comment:
Should this specifically call out Cloud Monitoring and Cloud Trace?

Reply:
Done. I've also made it so each may be enabled independently of the other.